### PR TITLE
update vdisktestcases

### PR DESCRIPTION
**Pull request by Dinaamagdy**

### CODE REVIEW

**Comment by Dinaamagdy** [comment]
```
dina@dina-Inspiron-5423:~/0-orch/vdisk/0-orchestrator/tests/0_orchestrator/test_suite$ nosetests -s -v testcases/basic_tests/test07_vms_apis.py  --tc-file=config.ini 
GAT-067 ... ok
GAT-068 ... ok
GAT-069 ... ok
GAT-070 ... ok
GAT-071 ... ok
GAT-072 ... ok
GAT-073 ... ok
GAT-075 ... ok
GAT-076 ... SKIP: https://github.com/zero-os/0-orchestrator/issues/994
GAT-077 ... SKIP: need at least 2 nodes

----------------------------------------------------------------------
Ran 10 tests in 1462.206s

OK (SKIP=2)
````

**Comment by Dinaamagdy** [comment]
```
dina@dina-Inspiron-5423:~/0-orch/vdisk/0-orchestrator/tests/0_orchestrator/test_suite$ nosetests -s -v testcases/basic_tests/test06_vdisks_apis.py  --tc-file=config.ini 
GAT-061 ... ok
GAT-062 ... ok
GAT-063 ... ok
GAT-064 ... ok
GAT-065 ... ok
GAT-066 ... SKIP: https://github.com/zero-os/0-orchestrator/issues/808
GAT-143 ... ok
GAT-144 ... SKIP: https://github.com/zero-os/0-orchestrator/issues/1148
GAT-145 ... ok
GAT-146 ... SKIP: https://github.com/zero-os/0-orchestrator/issues/808

----------------------------------------------------------------------
Ran 10 tests in 1089.464s

OK (SKIP=3)
```

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test06_vdisks_apis.py [hunk 1]
```diff
@@ -183,20 +191,106 @@ def test006_Rollback_vdisk(self):
...
+        response = self.vdisks_api.get_vdiskstorage_info(self.vdiskstoragedata["id"])
+        self.assertEqual(response.status_code, 200)
+        for key in self.vdiskstoragedata.keys():
+            if key in list(response.json().keys()):
+                self.assertEqual(self.vdiskstoragedata[key], response.json()[key])
+        self.lg.info(' [*] Get nonexisting vdiskstorage, should fail with 404')
+        response = self.vdisks_api.get_vdiskstorage_info(self.rand_str())
+        self.assertEqual(response.status_code, 404)
+
+    @unittest.skip('https://github.com/zero-os/0-orchestrator/issues/808')
```
Comment (islamTaha12) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test06_vdisks_apis.py]: This issue is closed

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py [hunk 2]
```diff
@@ -3,34 +3,35 @@
...
                 self.skipTest(' [*] No free disks to create storagecluster')
-    
-            self.lg.info(' [*] Deploy new storage cluster (SC0)')            
-            response, data = self.storageclusters_api.post_storageclusters(
-                nodes=nodes, 
-                driveType=disk_type, 
-                servers=random.randint(1, number_of_free_disks)
-            )
+            nodes = [self.nodeid]
+            numberOfDisks, diskType = max([(sum([1 for x in free_disks if x.get('type') == y]), y) for y in ['ssd', 'hdd', 'nvme']])
```
Comment (islamTaha12) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: number_of_disks

**Pull request merged**